lib/index.js

```diff
@@ -37,10 +38,9 @@ Amplitude.prototype.group = group;
 function track(payload, fn){
   var self = this;
   return this
-    .get('/httpapi')
-    .type('json')
-    .query({ api_key: this.settings.apiKey })
-    .query({ event: JSON.stringify(payload) })
+    .post('/httpapi')
+    .send('api_key=' + this.settings.apiKey)
```

Comment:
you should be able to just do `.send({ api_key: this.settings.apiKey, event: payload })` no?